packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -47,7 +47,20 @@ export 'package:matcher/expect.dart' hide expect, isInstanceOf;
 // that doesn't apply here.
 export 'package:test_api/hooks.dart' show TestFailure;
 export 'package:test_api/scaffolding.dart'
-    hide group, setUp, setUpAll, tearDown, tearDownAll, test;
+    show
+        OnPlatform,
```

Comment:
I went with the `dart format` formatting for a long list of shown members.

In `package:flutter` I mostly see a format with `show` on the same line as `import` and 2 spaces for the members.
In `package:flutter_tools` I mostly see a format like this, but a few use 2/4 space indent instead of 4/8.
In `package:flutter_test` There aren't other examples with enough members to split across lines.

Reply:
yeah in general if pushed i would say that any time you can't fit them on one line it's probably a mistake to be using show/hide anyway. :-) (This being one of the rare exceptions where it's reasonable, I think.)